### ISSUE TITLE
[question] How do I provide my own config file to Nextflow?

### ISSUE THREAD

**Issue by rpetit3** [bactopia/bactopia#86]
Nextflow allows you to provide a [Config file](https://www.nextflow.io/docs/latest/config.html) at run time. I have SGE, or my queues are named differently for SLURM, etc...

How would this be done in Bactopia? Is there a template?

**Comment by rpetit3** [comment]
This will be available in 1.3.1

**Comment by rpetit3** [comment]
[Bactopia v1.3.1](https://github.com/bactopia/bactopia/releases/tag/v1.3.1) added `--nfconfig` which allows the user to provide their own config file.

```
--nfconfig STR          A Nextflow compatible config file for custom profiles. This allows 
                                you to create profiles specific to your environment (e.g. SGE,
                                AWS, SLURM, etc...). This config file is loaded last and will 
                                overwrite existing variables if set.
                                Default: Bactopia's default configs
```
